applications/luci-app-p910nd/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2008-2014 The LuCI Team <luci@lists.subsignal.org>
+# Copyright (C) 2024 systemcrash
```

Comment:
Please supply a proper copyright header or none at all. „systemcrash“ is a too generic alias to ensure proper attribution. If you want to appear with an alias there then at least provide some form of contact info, e.g. a mail address.